<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>multipleGraphsWindow</class>
 <widget class="QWidget" name="multipleGraphsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multiple Graphs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="multipleGraphsWidget" name="wgt_graph" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_status_lbl">
     <property name="text">
      <string>status:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="lbl_status">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string>not running</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QPushButton" name="btn_startStop">
     <property name="text">
      <string>start drawing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QCheckBox" name="chb_setZeroAtStopSignal">
     <property name="text">
      <string>stop drawing at stop signal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QComboBox" name="cob_sensors"/>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="btn_clearData">
     <property name="text">
      <string>clear data</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QPushButton" name="btn_addSensor">
     <property name="text">
      <string>add sensor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="chb_normalize">
     <property name="text">
      <string>scale values to 0-1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QPushButton" name="btn_removeSensors">
     <property name="text">
      <string>remove selected sensors</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QLabel" name="lbl_description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Hint: Data from added sensors is saved, only NOT selected values are drawn.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="6">
    <widget class="QListWidget" name="list_sensors">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_axes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>y-axis:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QComboBox" name="cob_yAxis"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>multipleGraphsWidget</class>
   <extends>QWidget</extends>
   <header>multiplegraphswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
